crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -1,11 +1,13 @@
-use crate::prelude::{Children, Parent, PreviousParent};
+use smallvec::SmallVec;
+
 use bevy_ecs::{
     bundle::Bundle,
     entity::Entity,
     system::{Command, Commands, EntityCommands},
     world::{EntityMut, World},
 };
-use smallvec::SmallVec;
+
+use crate::prelude::{Children, Parent, PreviousParent};
```

Comment:
We really need automated tooling for our imports. It's unclear to me why these have moved, but possibly it's our style?

Reply:
I think CLion tries to organize imports as external, internal, crate. I can revert if desired.